st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -120,31 +122,15 @@ def connect(self):
                  False otherwise.
         :rtype: ``bool``
         """
-        conninfo = {'hostname': self.hostname,
-                    'port': self.port,
-                    'username': self.username,
-                    'allow_agent': False,
-                    'look_for_keys': False,
-                    'timeout': self.timeout}
-
-        if self.password:
-            conninfo['password'] = self.password
-
-        if self.key_files:
-            conninfo['key_filename'] = self.key_files
-
-        if self.key_material:
-            conninfo['pkey'] = self._get_pkey_object(key_material=self.key_material)
-
-        if not self.password and not (self.key_files or self.key_material):
-            conninfo['allow_agent'] = True
-            conninfo['look_for_keys'] = True
-
-        extra = {'_hostname': self.hostname, '_port': self.port,
-                 '_username': self.username, '_timeout': self.timeout}
-        self.logger.debug('Connecting to server', extra=extra)
-
-        self.client.connect(**conninfo)
+        if self.bastion_host:
+            self.logger.debug('Bastion host specified, connecting')
+            self.bastion_client = self._connect(self.bastion_host)
+            transport = self.bastion_client.get_transport()
+            real_addr = (self.hostname, self.port)
+            local_addr = ('', 0)
```

Comment:
If it's not too much hassle, could you please include the link to the fabric commit as a link? This might throw off other people when they read the code. Some helpful hinting would be nice IMO.